astroplan/tests/test_core.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
+import numpy as np
+import ephem
+import pytz
+
+from ..core import FixedTarget, Observer
+
+
+def test_Observer_constructor_location():
+    '''
+    Show that location defined by latitude/longitude/elevation is parsed
+    identically to passing in an `~astropy.coordinates.EarthLocation` directly.
+    '''
+
+    lat = '+19:00:00'
+    lon = '-155:00:00'
+    elevation = 0.0 * u.m
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+
+    environment_kwargs = dict(pressure=1*u.bar, relative_humidity=0.1,
+                              temperature=10*u.deg_C)
+
+    obs1 = Observer(name='Observatory',
+                    latitude=lat,
+                    longitude=lon,
+                    elevation=elevation,
+                    **environment_kwargs)
+
+    obs2 = Observer(name='Observatory',
+                    location=location,
+                    **environment_kwargs)
+
+    assert obs1.location == obs2.location, ('Locations not parsed equivalently')
```

Comment:
@adrn – I'm not sure I understand what you mean with "assert fails".
All asserts in tests should always be true, no?

I mean that here `, ('Locations not parsed equivalently')` can be removed because it's useless.